txwrapplog.py

```diff
@@ -65,6 +72,10 @@ def wrapped(event):
     return wrapped
 
 
+def _timestamp():
```

Comment:
I would suggest to use _timestamp as an internal function to the definition as it is not being used anywhere else ?
What do you think ?